doc/admin-guide/monitoring/statistics/core/network-io.en.rst

```diff
@@ -61,6 +61,7 @@ Network I/O
    :type: counter
 
 .. ts:stat:: global proxy.process.net.default_inactivity_timeout_applied integer
```

Comment:
Could you put in a few words about the difference between default_inactivity_timeout_applied and default_inactivity_timeout_count?

Reply:
👍 Added a note on each.